File: Web/RMA-Web-Shell/projects/claimcare/src/app/recovery-manager/shared/entities/claim-recovery-view.ts
import { RolePlayer } from 'projects/clientcare/src/app/policy-manager/shared/entities/roleplayer';
import { ClaimInvoice } from '../../../claim-manager/shared/entities/claim-invoice.model';

export interface ClaimRecoveryView {
    eventDescription: string;
    deceased: RolePlayer;
    recoveryRolePlayer: RolePlayer;
    insuredDeathDate: Date;
    claimNumber: number;
    claimReferenceNumber: string;
    claimCreatedDate: Date;
    claimInvoice: ClaimInvoice;
    claimId: number;
}

File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/ClaimRecoveryView.cs
using RMA.Service.ClientCare.Contracts.Entities.RolePlayer;

using System;

namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class ClaimRecoveryView
    {
        public string EventDescription { get; set; }
        public RolePlayer Deceased { get; set; }
        public RolePlayer RecoveryRolePlayer { get; set; }
        public DateTime InsuredDeathDate { get; set; }
        public int ClaimNumber { get; set; }
        public string ClaimReferenceNumber { get; set; }
        public DateTime ClaimCreatedDate { get; set; }
        public ClaimInvoice ClaimInvoice { get; set; }
        public int ClaimId { get; set; }
    }
}